# frame_type: 0x34(52)
# position ->  min: -134,217,728 max: 134,217,727
# 1 <= speed <= 2,500,000
int32 position
int32 speed